Tests/SCCAPIRequestTests.swift

```diff
@@ -136,7 +136,7 @@ class SCCAPIRequestTests: XCTestCase {
     func test_requestWithCallbackURL_validatesAmount() {
         let invalidAmounts:[SCCMoney] = [
             try! SCCMoney(amountCents: -100, currencyCode: "USD"),
-            try! SCCMoney(amountCents: 99, currencyCode: "USD"),
+            try! SCCMoney(amountCents: 00, currencyCode: "USD"),
```

Comment:
Should this crash? Will it crash?

Reply:
No, the iOS team has gone with a strict proposal of never crashing. So if you pass `0` it will return a throwable error that you can catch and get a detail of why it failed